### ISSUE TITLE
License

### ISSUE THREAD

**Issue by wallento** [fossi-foundation/wishbone#18]
I am wondering if it makes sense to consider putting a proper license on the document. In many jurisdictions it is impossible to waiver copyrights and the concept of public domain is defined differently.

CC0 seems like a good candidate but the question is what all authors intended and if we want/must track them down for consent. @rherveille what do you think?

**Comment by JimLewis** [comment]
CC BY-SA   - requires updates to be shared among the community.   

However, to be a standard - even a defacto standard, there needs to be some centralized group who owns integrating and resolving conflicting changes.   Which is how I explain a little about the copyright of B4.   

If you look at the old parallel port (mainly printers, but sometimes other devices), before IEEE 1284 the interface was a train wreck.   Hardware and software had to be configured for each different printer that was connected.   It was chaos and real ugly.

**Comment by ams** [comment]
> CC BY-SA - requires updates to be shared among the community.

No, it just requires that if you distribute such updates that they are licensed under the same license.  It does not require to share those modifications to anyone.

https://creativecommons.org/licenses/by-sa/4.0/deed
